pkg/ddc/base/log.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2021 The Fluid Authors.
```

Comment:
`2021` is OK, but `2022` in my opinion is better. This indicates when the file or the license header is added in this project.

LGTM

Reply:
Anyway, Much thanks for your contribution!